tollway/utils.py

```diff
@@ -34,34 +34,10 @@ def write_to_file(filename: str, events_log: list[Mapping[str, Union[str, bool]]
         json.dump(obj=events_log, fp=file, indent=1)
 
 
-def get_envs() -> dict:
-    envs = {}
+def get_envs(env: str) -> int:
+    defaults = {"duplicate": 50, "all": 250, "seconds": 10, "minutes": 20, "hours": 30, "days": 100}
 
-    duplicate_rate = config("DUPLICATE_RATE", default=50)
-    all_events_count = config("ALL_EVENTS_COUNT", default=250)
-    late_seconds_rate = config("LATE_SECONDS_RATE", default=10)
-    late_minutes_rate = config("LATE_MINUTES_RATE", default=20)
-    late_hours_rate = config("LATE_HOURS_RATE", default=30)
-    late_days_rate = config("LATE_DAYS_RATE", default=100)
-
-    if duplicate_rate == "":
-        duplicate_rate = 50
-    if all_events_count == "":
-        all_events_count = 250
-    if late_seconds_rate == "":
-        late_seconds_rate = 10
-    if late_minutes_rate == "":
-        late_minutes_rate = 20
-    if late_hours_rate == "":
-        late_hours_rate = 30
-    if late_days_rate == "":
-        late_days_rate = 100
-
-    envs["duplicate_rate"] = duplicate_rate
-    envs["all_events_count"] = all_events_count
-    envs["late_seconds_rate"] = late_seconds_rate
-    envs["late_minutes_rate"] = late_minutes_rate
-    envs["late_hours_rate"] = late_hours_rate
-    envs["late_days_rate"] = late_days_rate
-
-    return envs
+    env_variable = config(env.upper(), default=defaults[env])
```

Comment:
the only place I'm using `utils.get_env()` is in the `constants` module and I don't think it would with the way I'm currently using it. Might be a better way to implement.


<img width="293" alt="Screenshot 2024-02-09 at 5 38 24 PM" src="https://github.com/BenGriffith/tollway-traffic/assets/6089155/c3db06fb-3e86-4b15-a8a5-0981af14f5dd">

Reply:
ok all good then.